.github/workflows/notify-issue.yml

```diff
@@ -0,0 +1,20 @@
+name: Issue Slack Notifier
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send notification to Slack
+        uses: slackapi/slack-github-action@v1.24.0
+        with:
+          payload: |
+            {
+              "issue_url": "${{ github.event.issue.html_url }}"
```

Comment:
Should we include title?

Reply:
My thoughts were that it wasn't needed as Slack automatically embeds the issue title and initial content as part of it's link unfurling

Reply:
Indeed. I haven't though about that. It could be an issue with private PR, but we won't have such one.
Let's try without a title and have a look how slack will manage preview.